LanguageFeatures/Records/record_expressions_A02_t02.dart

```diff
@@ -33,22 +33,22 @@
 
 import "../../Utils/expect.dart";
 
-Record foo1() => (1,);
+(int,) foo1() => (1,);
```

Comment:
Oops, this one had been landed since I last loaded the page. ;-)

Reply:
> For single-positional record types, the form like (int) (with no trailing comma) is allowed as well, so we should test that it is supported.

@eernstg I don't think that it is supported. [Spec](https://github.com/dart-lang/language/blob/master/accepted/future-releases/records/records-feature-specification.md#record-type-annotations) reads

> It is a compile-time error if a record type has any of:
> ...
> - Only one positional field and no trailing comma. _This isn't ambiguous, since there are no parenthesized type expressions in Dart. But prohibiting this is symmetric with record expressions and leaves the potential for later support for parentheses for grouping in type expressions._

`record_type_annotations_A04_t01.dart` in particular checks that
```dart
  (int) foo() => (42);
//^^^^^
// [analyzer] unspecified
// [cfe] unspecified
```

And here we can improve the test changing it to:
```dart
  (int) foo() => (42,);
//^^^^^
// [analyzer] unspecified
// [cfe] unspecified
```
Probably it makes sense to have both of these checks